### PR TITLE
Add jquery ui doc

### PR DESCRIPTION
**Pull request by VincentLanglet**

### CODE REVIEW

**Comment by VincentLanglet** [comment]
@OskarStark @sonata-project/contributors I'm not familiar with this linter.
Does someone understand the error ?

**Review by jordisala1991 — DISMISSED** [review]

File: docs/cookbook/recipe_jquery_ui.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+The admin only comes with the `Jquery UI Sortable plugin`_.
+
+Adding another jQuery UI plugin
+-------------------------------
+
+To add another jQuery UI plugin in your admin, you have to tell
+webpack encore to not provide another instance of jQuery thanks
+to the `addExternals` method:
+
+.. code-block:: javascript
```
Comment (jordisala1991) [inline docs/cookbook/recipe_jquery_ui.rst:13]: maybe there is no lint configured for js?

Comment (VincentLanglet) [reply docs/cookbook/recipe_jquery_ui.rst:13]: There is already https://github.com/sonata-project/SonataAdminBundle/blame/4.x/docs/reference/troubleshooting.rst#L47

File: docs/cookbook/recipe_jquery_ui.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    # webpack.config.js
```
Comment (jordisala1991) [inline docs/cookbook/recipe_jquery_ui.rst]: can you use // instead? maybe is that

Comment (VincentLanglet) [reply docs/cookbook/recipe_jquery_ui.rst]: indeed

**Review by jordisala1991 — APPROVED** [review]

**Review by franmomu — APPROVED** [review]

**Pull request merged**